src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.scss

```diff
@@ -42,28 +43,32 @@
   width: 210px;
 }
 
+.moreInformation {
+  margin-top: 10px;
+}
+
 .downloadLink {
   grid-area: downloadLink;
   color: $blue;
-  width: 66px;
-  height: 16px;
-  text-align: right;
-  line-height: 1;
-
-  span {
-    cursor: pointer;
-  }
+  text-align: left;
+  line-height: 2;
```

Comment:
This looks fishy. I can't clearly explain why, but setting up distances between elements using line-height feels hacky (the reason for `line-height: 1` in the deleted code was to remove the extra height that the inherited line height of 1.4 would otherwise give to this text element). Wouldn't margin-bottom on `.moreInformation` produce a similar result?

Reply:
I am also not a fan of positioning using line-height, but the code was already using it so I just adjusted it. Will change it.